.github/workflows/sync-grapher-schema-to-digital-ocean.yml

```diff
@@ -0,0 +1,20 @@
+name: Upload to DO Spaces
+on:
+    push:
+        branches:
+            - master
+            - schema-do-sync-github-action
+jobs:
+    upload:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout Repository
+              uses: actions/checkout@master
+            - uses: BetaHuhn/do-spaces-action@v2
+              with:
+                  access_key: ${{ secrets.DO_ACCESS_KEY}}
+                  secret_key: ${{ secrets.DO_SECRET_KEY }}
+                  space_name: ${{ secrets.DO_SPACE_NAME }}
```

Comment:
hm, I feel like we should hardcode the space name and region here?
Otherwise, if we ever use more than on DO Spaces action (and multiple buckets), it becomes a mess to have multiple secrets hanging around in the repo.